### DISCUSSION TITLE
Sentry doesn't capture the websocket errors (for rest request is ok)

### DISCUSSION THREAD

**Discussion by mohammadranjbarz** [feathersjs/feathers#2406] [Q&A]
I added sentry to express with this codes ( as the sentry said)

```
const Sentry = require('@sentry/node');
const Tracing = require('@sentry/tracing');
Sentry.init({
  dsn: MY_DNS,
  integrations: [
    // enable HTTP calls tracing
    new Sentry.Integrations.Http({ tracing: true }),
    // enable Express.js middleware tracing
    new Tracing.Integrations.Express({ app }),
  ],

  // Set tracesSampleRate to 1.0 to capture 100%
  // of transactions for performance monitoring.
  // We recommend adjusting this value in production
  tracesSampleRate: 1.0,
});

// RequestHandler creates a separate execution context using domains, so that every
// transaction/span/breadcrumb is attached to its own Hub instance
app.use(Sentry.Handlers.requestHandler());
// TracingHandler creates a trace for every incoming request
app.use(Sentry.Handlers.tracingHandler());

// All controllers should live here
app.get("/", function rootHandler(req, res) {
  res.end("Hello world!");
});

// The error handler must be before any other error middleware and after all controllers
app.use(Sentry.Handlers.errorHandler());
```

### Expected behavior
I expect errors on websocket requests goes to sentry as in rest requests, but for websocket requests nothing happens

### Actual behavior
for REST requests it's ok,  and crash reports works fine, 

**Module versions** (especially the part that's not working):
    "@feathersjs/express": "^4.5.11",
    "@feathersjs/feathers": "^4.5.11",
    "@feathersjs/socketio": "^4.5.11",
    "@sentry/node": "^6.8.0",
    "@sentry/tracing": "^6.8.0",
**NodeJS version**:
10

**Comment by daffl** [comment]
This is what [Feathers hooks](https://docs.feathersjs.com/api/hooks.html) are for which run for both, REST and websocket calls. You can manually capture and send the events you need for all requests in an [application level hooks](https://docs.feathersjs.com/api/hooks.html#application-hooks).

**Reply by mohammadranjbarz** [reply]
Thanks it worked for me
